itest/universe_test.go

```diff
@@ -6,7 +6,10 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"fmt"
+	"github.com/lightninglabs/taproot-assets/asset"
```

Comment:
nit: Goland auto import issue. Can solve by adding an entry to "Settings -> Tools -> Actions on Save" that executes `gosimports`.